sound/soc/intel/boards/sdw_rt700.c

```diff
@@ -279,7 +279,7 @@ static int mc_probe(struct platform_device *pdev)
 	card->dev = &pdev->dev;
 
 	/* override platform name, if required */
-	mach = (&pdev->dev)->platform_data;
+	mach = pdev->dev.platform_data;
```

Comment:
same for all sdw parts, not upstream just yet. One patch per non-upstream driver would be real nice.